nsv13/code/modules/crew_objectives/engineering_objectives.dm

```diff
@@ -0,0 +1,76 @@
+/*				ENGINEERING OBJECTIVES				*/
+
+/datum/objective/crew/integrity //ported from old Hippie
```

Comment:
Why not include something about our overmap state here?

Reply:
uhhh,  what do you mean

Reply:
Hull Integrity, Armour quadrants etc.

Reply:
wait you're saying we dont already have that objective?